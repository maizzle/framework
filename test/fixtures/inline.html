<!DOCTYPE html>
<html>
<head>
  <if condition="page.css">
    <style>{{{ page.css }}}</style>
  </if>
</head>
<body>
  <table class="w-full cursor-move text-center">
    <tr>
      <td class="h-full align-middle">Test</td>
    </tr>
    <tr>
      <td class="px-2 py-4">
        <div class="mx-2 my-4">Test</div>
        <p class="mx-2 my-4">Test</p>
      </td>
    </tr>
  </table>
</body>
</html>
