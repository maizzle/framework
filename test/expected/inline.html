<!DOCTYPE html>
<html>
<head>

    <style>.w-full {
  width: 100% !important
}

.align-middle {
  vertical-align: middle !important
}

.text-center {
  text-align: center !important
}

.px-2 {
  padding-left: 0.5rem !important;
  padding-right: 0.5rem !important
}

.py-4 {
  padding-top: 1rem !important;
  padding-bottom: 1rem !important
}

.mx-2 {
  margin-left: 0.5rem !important;
  margin-right: 0.5rem !important
}

.my-4 {
  margin-top: 1rem !important;
  margin-bottom: 1rem !important
}

.h-full {
  height: 100% !important
}

.table {
  display: table !important
}

.cursor-move {
  cursor: move !important
}</style>
  </head>
<body>
  <table class="w-full cursor-move text-center" style="width: 100%; text-align: center;" width="100%" align="center" cellpadding="0" cellspacing="0" role="presentation">
    <tr>
      <td class="h-full align-middle" style="vertical-align: middle; height: 100%;" height="100%">Test</td>
    </tr>
    <tr>
      <td class="px-2 py-4" style="padding: 1rem 0.5rem;">
        <div class="mx-2 my-4" style="margin: 1rem 0.5rem;">Test</div>
        <p class="mx-2 my-4" style="margin-left: 0.5rem; margin-right: 0.5rem; margin-top: 1rem; margin-bottom: 1rem;">Test</p>
      </td>
    </tr>
  </table>
</body>
</html>
